### ISSUE TITLE
Document minimum Android version required

### ISSUE THREAD

**Issue by abartov** [openwallet-foundation-labs/learner-credential-wallet#660]
**Describe the bug**
The app cannot be installed on any of my (admittedly not-cutting-edge) Android devices, but Google Play does not specify what version is required. It would be helpful to understand what the minimum Android version required is, but I could not find mention of it on the DCC public pages or in the README file.

**Smartphone (please complete the following information):**
 - OS: Android 10

**Comment by alexfigtree** [comment]
@abartov Would it be possible for you upgrade your Android OS to the latest version (Android 14 in most devices and 15 in some) and let us know if you continue to run into issues?

**Comment by abartov** [comment]
No, it is not. This is an older device, and Android 10 is the latest supported version for it. My concern is precisely that many of our prospective users would be using similar older devices, causing a significant gap in the ability to obtain credentials.

**Comment by dmitrizagidulin** [comment]
@abartov yeah, very valid concern, and unfortunately there's nothing we can do about it. This past year, Google has dictated that all active apps in their store need to update to Android 14 (see https://developer.android.com/google/play/requirements/target-sdk ). So this is going to be the case for pretty much all mobile wallets.

(You're right that we should document the version requirement)

**Comment by abartov** [comment]
Oh, I didn't know about the Google requirement!  That is indeed unfortunate, and does mean that wallet-based solutions won't be available for people with devices that are more than three years old. Thank you for clarifying that.

**Comment by alexfigtree** [comment]
@abartov As you may have seen in another repository issue, we've downgraded the minSdkVersion in PR #662, so you should now be able to install it on devices running Android. I've tested it out on an Android 10 simulator and was able to install successfully, but not on an actual hardware device. Please download the latest version [here](https://play.google.com/store/apps/details?id=app.lcw) on your device and let us know if you run into any issues. Supported versions and target levels can be seen in app.json file at the root of our project.

**Comment by abartov** [comment]
Thank you! I was able to install the app on an Android 10 phone. Running it, I authorized its permissions request, and then tried to register the wallet as prompted, but nothing happened after the permission approval. Further clicking on the Register button likewise did nothing.
Are there sample credentials we can further test with? My wallet is associated with the email address **asaf.bartov@gmail.com**.

**Comment by alexfigtree** [comment]
@abartov We have mock credentials for devs to try out on the app. You can access them by going to the Settings Icon (lower right hand corner) --> About --> Click on version (ex: v2.1.5 - Build 85) --> Add mock credentials

**Comment by kayaelle** [comment]
Hi @abartov - we're going to pull it out of the install flow for now. It will continue to be available in the settings while we work on the user experience. It does register the wallet with a browser but doesn't provide feedback when the registration is complete. We'll rework it. Thanks!
